packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -197,6 +247,42 @@ class HorizontalOverflow extends Foundation<
                 itemsHeight,
             });
         }
+
+        this.handleOverflowChange();
```

Comment:
This assumes that the overflow is changing everytime an item is loaded, there should be a check to see if the overflow has actually changed.